app/boards/makerbase/mks_canable_v20/mks_canable_v20.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <zephyr/dt-bindings/gpio/gpio.h>
+
+/ {
+	cannectivity: cannectivity {
+		compatible = "cannectivity";
+		timestamp-counter = <&counters2>;
+
+		channel0 {
+			compatible = "cannectivity-channel";
+			can-controller = <&fdcan1>;
+			state-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;  /* blue led */
+			activity-gpios = <&gpioa 0 GPIO_ACTIVE_LOW>;  /* green led */
+		};
+	};
+};
+
+&zephyr_udc0 {
+	gs_usb0: gs_usb0 {
+		compatible = "gs_usb";
+	};
+};
+
+&timers2 {
+	/* Timer with 1us tick used for CAN timestamp */
```

Comment:
Please remove this comment. The purpose is obvious.

Reply:
Could we please leave it. Reading this dts file first time does not give any clue, how prescaler is related to the tick value of 1us.  Just an example of typical determining of the tick value:

1. Without comment:

- Find PLL clock for timer (need to look in board's main device  tree)
- Divide PLL clock by prescaler + 1 (need to look up st's reference manual)
In total: x2 time consuming steps are needed

3. With comment:

- Take a look on comment and see value directly.

In total: x1 very simple step needed

Reply:
My point is, that the CANnectivity timestamp counter _always_ runs @ 1MHz. The comment does not add any value. 

Please remove it.

Reply:
Removed